src/annotations/components/annotationForm/AnnotationTimeInput.tsx

```diff
@@ -78,18 +80,20 @@ export const AnnotationTimeInput: FC<Props> = (props: Props) => {
     if (isValidTimeFormat(event.target.value)) {
       if (timeZone === 'UTC') {
         props.onChange(
-          moment
-            .utc(event.target.value, timeFormat)
-            .toDate()
-            .toISOString()
+          DateTime.fromFormat(
```

Comment:
It made sense to use `luxon's` in built `fromFormat` here than writing our own custom `fromFormat`.